src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
-import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div
+        className="card"
```

Comment:
please, use semantic tags, here ul, and li inside MovieCard component. pass key directly in <MovieCard... call